config/jobs/kubernetes-sigs/kueue/kueue-presubmits.yaml

```diff
@@ -33,6 +33,21 @@ presubmits:
         - make
         args:
         - test-integration
+  - name: pull-kueue-test-e2e-main
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/kueue
+    annotations:
+      testgrid-dashboards: sig-scheduling
+      testgrid-tab-name: pull-kueue-test-e2e-main
+      description: "Run kueue end to end tests"
+    spec:
+      containers:
+      - image: golang:1.18
```

Comment:
Are you sure this image supports creating a kind cluster?

Reply:
Changed.  I wasn't really sure if this would work but I took an example from the descheduler e2e tests.  Now that this is merged, I will see how to test this to verify everything is working.